src/Analyzer.Utilities/Extensions/UriExtensions.cs

```diff
@@ -0,0 +1,77 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Threading;
+using Microsoft.CodeAnalysis;
+
+namespace Analyzer.Utilities
+{
+    public static class UriExtensions
+    {
+        private static readonly ImmutableHashSet<string> s_uriWords = ImmutableHashSet.Create(StringComparer.OrdinalIgnoreCase, "uri", "urn", "url");
```

Comment:
https://github.com/dotnet/roslyn-analyzers/issues/3009 - this is the problem inclusion I think